File: test/demo/wslc-polyglot-hello/polyglot/servers/srv.go
package main
import("net";"os";"strconv")
func main(){ p,_:=strconv.Atoi(os.Args[1]); l,_:=net.Listen("tcp","0.0.0.0:"+strconv.Itoa(p)); c,_:=l.Accept(); b:=make([]byte,64); c.Read(b); c.Write([]byte("hello world from [go]\n")); c.Read(b); c.Close() }

File: test/demo/wslc-polyglot-hello/polyglot/servers/srv.rs
use std::net::TcpListener; use std::io::{Read,Write}; use std::env;
fn main(){ let p:u16=env::args().nth(1).unwrap().parse().unwrap();
  let l=TcpListener::bind(("0.0.0.0",p)).unwrap(); let (mut c,_)=l.accept().unwrap();
  let mut b=[0u8;64]; let _=c.read(&mut b); c.write_all(b"hello world from [rust]\n").unwrap(); let _=c.read(&mut b); }
